### PR TITLE
Finalized things with Loudness

### LINKED ISSUE #6562: Loudness: Pre-launch: Screenshot, readme.txt & description

### PR DESCRIPTION
**Pull request by pbking**
This change wraps up a few things to finalize Loudness.

With this I believe it is ready for release to wporg.

* Adds a screenshot
* Adds a description
* Updates version to 1.0
* Removes some unused assets (a font and a pattern)
* Renamed a few functions from the original "Decibel" to "Loudness"
* Removed a reference to the "landing page" template (which was converted to be "home.html" instead)

### CODE REVIEW

**Review by vcanales — APPROVED** [review]
Gave it a go, and it LGTM!

File: loudness/readme.txt [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-Tested up to: 6.0
+Tested up to: 6.1
 Requires PHP: 5.7
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 == Description ==
 
-A theme for music-y stuff
+A bold opinionated theme for music and learning
```
Comment (jffng) [inline loudness/readme.txt]:
```suggestion
A bold, opinionated theme for music and learning.
```

File: loudness/theme.json [hunk 2]
```diff
@@ -75,6 +68,7 @@
 								"file:./assets/fonts/DMMono-Regular.ttf"
 							]
 						}
+
```
Comment (jffng) [inline loudness/theme.json]:
```suggestion
```

**Review by jffng — APPROVED** [review]
Pre-approving with a couple nits, ty!

**Pull request merged**